From dee94d832ae3a9c862370ddbe07a9fd01ce19d7f Mon Sep 17 00:00:00 2001
From: Peter O'Connor <peter@solus-project.com>
Date: Thu, 14 Jun 2018 16:39:08 +1000
Subject: [PATCH] Hardcode /usr/lib64 path for locale

This enables a single copy to be used for 32bit builds.

Signed-off-by: Peter O'Connor <peter@solus-project.com>
---
 Makeconfig | 4 +---
 1 file changed, 1 insertions(+), 3 deletions(-)

diff --git a/Makeconfig b/Makeconfig
index 1afe86475c..1a029a0148 100644
--- a/Makeconfig
+++ b/Makeconfig
@@ -177,9 +177,7 @@ inst_zonedir = $(install_root)$(zonedir)
 
 # Where to install the compiled binary locale archive and compiled
 # binary locale files.
-ifndef complocaledir
-complocaledir = $(libdir)/locale
-endif
+complocaledir = /usr/lib64/locale
 inst_complocaledir = $(install_root)$(complocaledir)
 
 # Where to install the message catalog data files (which are
--
2.17.1
